app/views/layouts/application.html.erb

```diff
@@ -17,7 +17,13 @@
         <div class="menu-item">
           <%= menu_link_to 'All users', users_path %>
         </div>
+        <% if current_user %>
+          <div class="menu-item">
+            <%= link_to 'My profile', current_user %>
+          </div>
+        <% end %>
```

Comment:
- [ ] Kindly move this logic into helpers

Reply:
@oloomoses The `application.html.erb` file was inherited from the `Microverse ror-social-scaffold` repo. Lines 23 to 38 contain logic specific to signing in a user, signing out a user, notices and alerts. We added lines 20 - 24  to render content from the `#usershow` in the controller. The request to move this logic to helpers means previous work on this repo was not only incorrectly done but ignores the fact that the logic here calls a controller action.